packages/widget/src/stores/form/useFormRef.ts

```diff
@@ -0,0 +1,66 @@
+import { useImperativeHandle } from 'react';
+import { useWidgetConfig } from '../../providers/WidgetProvider/WidgetProvider.js';
+import { useBookmarkActions } from '../../stores/bookmarks/useBookmarkActions.js';
+import { formDefaultValues } from '../../stores/form/createFormStore.js';
+import { useSendToWalletStore } from '../../stores/settings/useSendToWalletStore.js';
+import type { FormRef } from '../../types/widget.js';
+import { HiddenUI } from '../../types/widget.js';
+import type { FormValuesState, GenericFormValue } from './types.js';
+
+export const useFormRef = (formStore: FormValuesState, formRef?: FormRef) => {
+  const { setSendToWallet } = useSendToWalletStore();
```

Comment:
Can we select only `setSendToWallet` from the store to avoid re-renders when other store values change?

Reply:
See what you mean. changing that to useSendToWalletActions rather than useSendToWalletStore